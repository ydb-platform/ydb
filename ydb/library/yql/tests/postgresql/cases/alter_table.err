<sql-statement>
--
-- ALTER_TABLE
--
-- Clean up in case a prior regression run failed
SET client_min_messages TO 'warning';
</sql-statement>
<sql-statement>
DROP ROLE IF EXISTS regress_alter_table_user1;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 287
    	DROP ROLE IF EXISTS regress_alter_table_user1;
	^
<sql-statement>
RESET client_min_messages;
</sql-statement>
<sql-statement>
CREATE USER regress_alter_table_user1;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 285
    	CREATE USER regress_alter_table_user1;
	^
<sql-statement>
--
-- add attribute
--
CREATE TABLE attmp (initial int4);
</sql-statement>
<sql-statement>
COMMENT ON TABLE attmp_wrong IS 'table comment';
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 258
    	COMMENT ON TABLE attmp_wrong IS 'table comment';
	^
<sql-statement>
COMMENT ON TABLE attmp IS 'table comment';
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 258
    	COMMENT ON TABLE attmp IS 'table comment';
	^
<sql-statement>
COMMENT ON TABLE attmp IS NULL;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 258
    	COMMENT ON TABLE attmp IS NULL;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN xmin integer; -- fails
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN xmin integer; -- fails
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN a int4 default 3;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN a int4 default 3;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN b name;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN b name;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN c text;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN c text;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN d float8;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN d float8;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN e float4;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN e float4;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN f int2;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN f int2;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN g polygon;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN g polygon;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN i char;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN i char;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN k int4;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN k int4;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN l tid;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN l tid;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN m xid;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN m xid;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN n oidvector;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN n oidvector;
	^
<sql-statement>
--ALTER TABLE attmp ADD COLUMN o lock;
ALTER TABLE attmp ADD COLUMN p boolean;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	--ALTER TABLE attmp ADD COLUMN o lock;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN q point;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN q point;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN r lseg;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN r lseg;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN s path;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN s path;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN t box;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN t box;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN v timestamp;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN v timestamp;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN w interval;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN w interval;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN x float8[];
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN x float8[];
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN y float4[];
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN y float4[];
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN z int2[];
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN z int2[];
	^
<sql-statement>
INSERT INTO attmp (a, b, c, d, e, f, g,    i,    k, l, m, n, p, q, r, s, t,
	v, w, x, y, z)
   VALUES (4, 'name', 'text', 4.1, 4.1, 2, '(4.1,4.1,3.1,3.1)',
	'c',
	314159, '(1,1)', '512',
	'1 2 3 4 5 6 7 8', true, '(1.1,1.1)', '(4.1,4.1,3.1,3.1)',
	'(0,2,4.1,4.1,3.1,3.1)', '(4.1,4.1,3.1,3.1)',
	'epoch', '01:00:10', '{1.0,2.0,3.0,4.0}', '{1.0,2.0,3.0,4.0}', '{1,2,3,4}');
</sql-statement>
<sql-statement>
SELECT * FROM attmp;
</sql-statement>
<sql-statement>
DROP TABLE attmp;
</sql-statement>
<sql-statement>
-- the wolf bug - schema mods caused inconsistent row descriptors
CREATE TABLE attmp (
	initial 	int4
);
</sql-statement>
<sql-statement>
ALTER TABLE attmp ADD COLUMN a int4;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN a int4;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN b name;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN b name;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN c text;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN c text;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN d float8;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN d float8;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN e float4;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN e float4;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN f int2;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN f int2;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN g polygon;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN g polygon;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN i char;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN i char;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN k int4;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN k int4;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN l tid;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN l tid;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN m xid;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN m xid;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN n oidvector;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN n oidvector;
	^
<sql-statement>
--ALTER TABLE attmp ADD COLUMN o lock;
ALTER TABLE attmp ADD COLUMN p boolean;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	--ALTER TABLE attmp ADD COLUMN o lock;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN q point;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN q point;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN r lseg;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN r lseg;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN s path;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN s path;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN t box;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN t box;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN v timestamp;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN v timestamp;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN w interval;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN w interval;
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN x float8[];
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN x float8[];
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN y float4[];
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN y float4[];
	^
<sql-statement>
ALTER TABLE attmp ADD COLUMN z int2[];
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE attmp ADD COLUMN z int2[];
	^
<sql-statement>
INSERT INTO attmp (a, b, c, d, e, f, g,    i,   k, l, m, n, p, q, r, s, t,
	v, w, x, y, z)
   VALUES (4, 'name', 'text', 4.1, 4.1, 2, '(4.1,4.1,3.1,3.1)',
        'c',
	314159, '(1,1)', '512',
	'1 2 3 4 5 6 7 8', true, '(1.1,1.1)', '(4.1,4.1,3.1,3.1)',
	'(0,2,4.1,4.1,3.1,3.1)', '(4.1,4.1,3.1,3.1)',
	'epoch', '01:00:10', '{1.0,2.0,3.0,4.0}', '{1.0,2.0,3.0,4.0}', '{1,2,3,4}');
</sql-statement>
<sql-statement>
SELECT * FROM attmp;
</sql-statement>
<sql-statement>
CREATE INDEX attmp_idx ON attmp (a, (d + e), b);
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: index expression is not supported yet
    	CREATE INDEX attmp_idx ON attmp (a, (d + e), b);
	^
<sql-statement>
ALTER INDEX attmp_idx ALTER COLUMN 0 SET STATISTICS 1000;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:37: Error: ERROR:  column number must be in range from 1 to 32767

    	ALTER INDEX attmp_idx ALTER COLUMN 0 SET STATISTICS 1000;
	                                    ^
<sql-statement>
ALTER INDEX attmp_idx ALTER COLUMN 1 SET STATISTICS 1000;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER INDEX attmp_idx ALTER COLUMN 1 SET STATISTICS 1000;
	^
<sql-statement>
ALTER INDEX attmp_idx ALTER COLUMN 2 SET STATISTICS 1000;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER INDEX attmp_idx ALTER COLUMN 2 SET STATISTICS 1000;
	^
<sql-statement>
\d+ attmp_idx
</sql-statement>
Metacommand \d+ attmp_idx is not supported
<sql-statement>
ALTER INDEX attmp_idx ALTER COLUMN 3 SET STATISTICS 1000;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER INDEX attmp_idx ALTER COLUMN 3 SET STATISTICS 1000;
	^
<sql-statement>
ALTER INDEX attmp_idx ALTER COLUMN 4 SET STATISTICS 1000;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER INDEX attmp_idx ALTER COLUMN 4 SET STATISTICS 1000;
	^
<sql-statement>
ALTER INDEX attmp_idx ALTER COLUMN 2 SET STATISTICS -1;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER INDEX attmp_idx ALTER COLUMN 2 SET STATISTICS -1;
	^
<sql-statement>
DROP TABLE attmp;
</sql-statement>
<sql-statement>
-- fails with incorrect object type
CREATE VIEW at_v1 AS SELECT 1 as a;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: expected at least one target column
    	-- fails with incorrect object type
	^
<sql-statement>
ALTER TABLE at_v1 ALTER COLUMN a SET STATISTICS 0;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: RawStmt: alternative is not implemented yet : 244
    	ALTER TABLE at_v1 ALTER COLUMN a SET STATISTICS 0;
	^
<sql-statement>
DROP VIEW at_v1;
</sql-statement>
-stdin-:<main>: Error: Parse Sql

    -stdin-:<main>:1:1: Error: View not found: 'at_v1'
    	DROP VIEW at_v1;
	^
<sql-statement>
--
-- rename - check on both non-temp and temp tables
--
CREATE TABLE attmp (regtable int);
</sql-statement>
<sql-statement>
CREATE TEMP TABLE attmp (attmptable int);
</sql-statement>
(TFileError) (File exists) util/system/file.cpp:857: can't open "/tmp/tmpuixf4gua/attmp" with mode CreateNew (0x00000003)
