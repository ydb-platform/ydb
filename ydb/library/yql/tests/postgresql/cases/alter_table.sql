--
-- ALTER_TABLE
--
-- Clean up in case a prior regression run failed
SET client_min_messages TO 'warning';
RESET client_min_messages;
--
-- add attribute
--
CREATE TABLE attmp (initial int4);
INSERT INTO attmp (a, b, c, d, e, f, g,    i,    k, l, m, n, p, q, r, s, t,
	v, w, x, y, z)
   VALUES (4, 'name', 'text', 4.1, 4.1, 2, '(4.1,4.1,3.1,3.1)',
	'c',
	314159, '(1,1)', '512',
	'1 2 3 4 5 6 7 8', true, '(1.1,1.1)', '(4.1,4.1,3.1,3.1)',
	'(0,2,4.1,4.1,3.1,3.1)', '(4.1,4.1,3.1,3.1)',
	'epoch', '01:00:10', '{1.0,2.0,3.0,4.0}', '{1.0,2.0,3.0,4.0}', '{1,2,3,4}');
