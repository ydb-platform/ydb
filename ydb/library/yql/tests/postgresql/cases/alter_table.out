--
-- ALTER_TABLE
--
-- Clean up in case a prior regression run failed
SET client_min_messages TO 'warning';
RESET client_min_messages;
--
-- add attribute
--
CREATE TABLE attmp (initial int4);
INSERT INTO attmp (a, b, c, d, e, f, g,    i,    k, l, m, n, p, q, r, s, t,
	v, w, x, y, z)
   VALUES (4, 'name', 'text', 4.1, 4.1, 2, '(4.1,4.1,3.1,3.1)',
	'c',
	314159, '(1,1)', '512',
	'1 2 3 4 5 6 7 8', true, '(1.1,1.1)', '(4.1,4.1,3.1,3.1)',
	'(0,2,4.1,4.1,3.1,3.1)', '(4.1,4.1,3.1,3.1)',
	'epoch', '01:00:10', '{1.0,2.0,3.0,4.0}', '{1.0,2.0,3.0,4.0}', '{1,2,3,4}');
DROP TABLE attmp;
-- the wolf bug - schema mods caused inconsistent row descriptors
CREATE TABLE attmp (
	initial 	int4
);
INSERT INTO attmp (a, b, c, d, e, f, g,    i,   k, l, m, n, p, q, r, s, t,
	v, w, x, y, z)
   VALUES (4, 'name', 'text', 4.1, 4.1, 2, '(4.1,4.1,3.1,3.1)',
        'c',
	314159, '(1,1)', '512',
	'1 2 3 4 5 6 7 8', true, '(1.1,1.1)', '(4.1,4.1,3.1,3.1)',
	'(0,2,4.1,4.1,3.1,3.1)', '(4.1,4.1,3.1,3.1)',
	'epoch', '01:00:10', '{1.0,2.0,3.0,4.0}', '{1.0,2.0,3.0,4.0}', '{1,2,3,4}');
ALTER INDEX attmp_idx ALTER COLUMN 0 SET STATISTICS 1000;
ERROR:  column number must be in range from 1 to 32767
LINE 1: ALTER INDEX attmp_idx ALTER COLUMN 0 SET STATISTICS 1000;
                                           ^
DROP TABLE attmp;
--
-- rename - check on both non-temp and temp tables
--
CREATE TABLE attmp (regtable int);
